src/components/Body.js

```diff
@@ -1,24 +1,29 @@
 import { createBrowserRouter, RouterProvider } from "react-router-dom";
-import Browse from "./Browse";
-import Login from "./Login";
-import Movie from "./Movie";
-import StarredMovies from "./StarredMovies";
+import { lazy, Suspense } from "react";
+
+const Login  = lazy(() => import("./Login"));
+const Browse = lazy(() => import("./Browse"));
+const Movie  = lazy(() => import("./Movie"));
+const StarredMovies = lazy(() => import("./StarredMovies"));
+const Shimmer = lazy(() => import("./Shimmer"));
+
+
 const Body = () => {
   const AppRouter = createBrowserRouter([
     {
       path: "/",
-      element: <Login />,
+      element: <Suspense fallback={<Shimmer />}><Login /></Suspense>,
```

Comment:
you can make lazy loading to all pages 
it need to be added only , pages which are not used frequently

Reply:
okay making the required changes